### DISCUSSION TITLE
(PRO) Version of Installed RustDesk in WebConsole

### DISCUSSION THREAD

**Discussion by MikeT-KBU** [rustdesk/rustdesk-server-pro#441] [Q&A]
It would be nice to see in "Devices" Section the installed RustDesk Version.
Best would be in RustDesk Client in AdressBook to so one kann see where he needs to Update it.

**Comment by rustdesk** [comment]
dup: https://github.com/rustdesk/rustdesk-server-pro/issues/322

**Reply by MikeT-KBU** [reply]
Ok thx. So it is asked and ignored for over a year....
The Client infos are going over the API already so why can't you show it in WebConsole?
